src/log_outgoing_requests/tests/test_logging.py

```diff
@@ -63,12 +61,33 @@ def test_expected_data_is_saved_when_saving_enabled(self, m):
                 self.assertEqual(request_log.req_content_type, "")
                 self.assertEqual(request_log.res_content_type, "")
                 self.assertEqual(request_log.response_ms, 0)
+                self.assertEqual(
+                    request_log.req_headers,
+                    (
+                        "{'User-Agent': 'python-requests/2.26.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive'}"
+                        if method == "HEAD"
+                        else "{'User-Agent': 'python-requests/2.26.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '0'}"
```

Comment:
This is a bit much for an if/else expression, and blacks pretty ugly.

Also I'm not convinced this is a good format to store the headers in: if you want to keep a structure then it'd be JSON (and a JSONfield), not this Python repr() format (and without linebreaks it would not be very nice for debugging).

Would it be more useful to store them the way they are in HTTP? Just name/value pairs each on a line? Do we need all these braces and quotes?